src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql

```diff
@@ -0,0 +1,79 @@
+DECLARE cohort_date DATE DEFAULT '2020-03-01';
+DECLARE activation_period_start DATE DEFAULT '2020-03-02';
+DECLARE activation_period_end DATE DEFAULT '2020-03-07';
+/*DECLARE os_name STRING DEFAULT 'Android';
+DECLARE app STRING DEFAULT 'Fennec';*/
+
+WITH last_seen_data as (
+SELECT
+  submission_date,
+  client_id,
+  days_since_seen,
+  days_since_created_profile,
+  profile_date,
+  app_name,
+  SPLIT(metadata_app_version,'.')[offset(0)] as app_version,
+  os,
```

Comment:
@magorlick Not for this week but before we productionize next week. Let me know if there are any additional dimensions we'd like to add here.